lib/IRGen/ConstantBuilder.h

```diff
@@ -94,6 +94,15 @@ class ConstantAggregateBuilderBase
                             unsigned(reference.isIndirect()));
   }
 
+  /// Add an indirect relative reference to the given address.
+  /// The target must be a "GOT-equivalent", i.e. a pointer to an
+  /// external object.
+  void addIndirectRelativeAddress(ConstantReference reference) {
+    assert(reference.isIndirect());
+    addRelativeOffset(IGM().RelativeAddressTy,
+                      reference.getValue());
+  }
```

Comment:
There's already an overload of `addRelativeAddress` that takes a `ConstantReference` and does this.